### PR TITLE
mgr/cephadm: include alerts in prometheus deployment

### PR DESCRIPTION
**Pull request by liewegas**
- drop alerts file into place in prom container
- include alerts in all ceph rpms, not just the suse one

### CODE REVIEW

**Comment by liewegas** [comment]
@p-se note that this changes the location that was previous used in the ceph-grafana-dashboards package.  is that okay?  nothing relied on the old location?

**Comment by p-se** [comment]
> @p-se note that this changes the location that was previous used in the ceph-grafana-dashboards package. is that okay? nothing relied on the old location?

I'm not sure if the previous location has already been used somewhere, but as the change makes sense to me, we should have it merged and adapt anything required.

**Comment by p-se** [comment]
@swiftgist can you foresee any issues with this change?

File: debian/control [hunk 1]
```diff
@@ -1161,4 +1161,14 @@ Description: grafana dashboards for the ceph dashboard
...
+
+Package: ceph-prometheus-alerts
+Architecture: all
+Description: prometheus alerts for the ceph dashboard
+ Ceph is a massively scalable, open-source, distributed
+ storage system that runs on commodity hardware and delivers object,
+ block and file system storage.
+ .
+ This package contains alerts used for prometheus to interact with the
+ Ceph Dashboard.
```
Comment (sebastian-philipp) [inline debian/control]: is this still true? what about cephadm?

Comment (liewegas) [reply debian/control]: i don't think it directly relates to cephadm at all...

**Comment by swiftgist** [comment]
> @swiftgist can you foresee any issues with this change?

No issues, just the comment that ceph_default_alerts.yml isn't used by cephadm for container deployment or adoption.

**Pull request merged**